src/test/java/org/openmrs/performance/http/DoctorHttpService.java

```diff
@@ -347,4 +359,37 @@ public HttpRequestActionBuilder saveDiagnosis(String patientUuid, String encount
 		}
 	}
 
+	public HttpRequestActionBuilder saveVitalsData(String patientUuid) {
+		ZonedDateTime now = ZonedDateTime.now();
+		String encounterDatetime = now.format(DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSZ"));
+		
+		Map<String, Object> vitals = new HashMap<>();
+		vitals.put("form", VITALS_FORM_UUID);
+		vitals.put("patient", patientUuid);
+		vitals.put("location", VITALS_LOCATION_UUID);
+		vitals.put("encounterType", VITALS_ENCOUNTER_TYPE_UUID);
+		vitals.put("encounterDatetime", encounterDatetime);
+		
+		List<Map<String, Object>> obs = new ArrayList<>();
+		obs.add(Map.of("concept", SYSTOLIC_BLOOD_PRESSURE, "value", 34));
+		obs.add(Map.of("concept", DIASTOLIC_BLOOD_PRESSURE, "value", 44));
+		obs.add(Map.of("concept", RESPIRATORY_RATE, "value", 100));
+		obs.add(Map.of("concept", ARTERIAL_BLOOD_OXYGEN_SATURATION, "value", 20));
+		obs.add(Map.of("concept", PULSE, "value", 120));
+		obs.add(Map.of("concept", TEMPERATURE_C, "value", 28));
+		obs.add(Map.of("concept", WEIGHT_KG, "value", 60));
+		obs.add(Map.of("concept", HEIGHT_CM, "value", 121));
+		obs.add(Map.of("concept", MID_UPPER_ARM_CIRCUMFERENCE, "value", 34));
+		
+		vitals.put("obs", obs);
```

Comment:
Minor naming suggestion: `vitals` is a Map, not a collection, so the plural form might be misleading. We can rename it to `encounter` or `payload` or something similar.

`obs` could be renamed to `observations`.